package.json

```
@@ -1,7 +1,19 @@
 {
-  "name": "app",
-  "version": "0.1.0",
+  "name": "fashion24-front-end",
+  "version": "0.0.1",
   "private": true,
+  "license": "MIT",
+  "description": "",
+  "contributors": [
+    "JaeJin Lee <jinjinstar3@naver.com>"
```

```suggestion
    "JaeJin Lee <jinjinstar3@naver.com>"
    "new Name <newEmail@email.com>"
```
이런식으로 제안 걸어주시면 바로 해드릴게요